css/lesson.css

```diff
@@ -1,47 +1,126 @@
-/* Comments in code. */
-.comment {
-    color: purple;
+code {
+    color: #333333;
 }
 
-/* Highlighted changes in code. */
-.highlight {
-    background-color: mistyrose;
+/* Code sample */
+pre.sourceCode,
+pre.input {
+    color: ForestGreen;
 }
-
-/* Manual input. */
-.in {
-    color: darkgreen;
+pre.output {
+    color: MediumBlue;
 }
-
-/* Program output. */
-.out {
-    color: darkblue;
-    font-style: italic;
+pre.error {
+    color: Red;
 }
 
-/* Error output. */
-.err {
-    color: darkred;
-    font-style: italic;
-    font-weight: bold;
+.objectives, .keypoints, .callout, .challenge {
+    margin: 1em 0;
+    padding: 0em 1em;
 }
 
-/* Explanatory call-out boxes. */
-div.box {
-    background-color: lightgray;
+.objectives h2:first-child,
+.keypoints h2:first-child,
+.callout h2:first-child,
+.challenge h2:first-child {
+    margin-top: 10px;
 }
 
-/* Challenge questions. */
-div.challenges {}
+.objectives, .keypoints {
+    background-color: azure;
+    border: 5px solid azure;
+}
 
-/* Key points in summary. */
-div.keypoints {}
+.callout {
+    background-color: #EEE;
+    border: 5px solid #EEE;
+}
 
-/* Main lesson. */
-div.lesson {}
+.challenge {
+    background-color: #CCFFCC;
+    border: 5px solid #CCFFCC;
+}
 
-/* Learning objectives. */
-div.objectives {}
+/* Tables used for displaying choices in challenges. */
+table.choices tr td {
+    vertical-align : top;
+}
 
-/* Continuation paragraph. */
-p.continue {}
+/* Database tables do _not_ have double borders */
+table.outlined {
+    border-collapse: collapse;
+}
+    
+/* Printing */
+@media print {
```

Comment:
It looks like most of the elements affected are only in lessons anyway. The exceptions are h1,h2,h3,h4,h5,h6,p,ul,ol,li,pre,code, which may also affect other things globally in site and workshop.

Presumably the print styles for these elements in lesson and workshop should be the same anyway? Is their a maintained print style for site? It seems less important than the print style for lessons and workshops.

Do you want me to remove or fix this in some way before merging the PR?